### PR TITLE
global-banner: change to layout and documentation

### PR DESCRIPTION
**Pull request by davidpauljunior**
- Removes pseudo elements that were previously used to extend the banner to the edge of the screen
- Updates the documentation with information on using app level containers to control the content
- Adds horizontal padding and a flush modifier to allow overriding of it

### CODE REVIEW

File: toolkits/global/packages/global-banner/package.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
   "name": "@springernature/global-banner",
-  "version": "0.1.1",
+  "version": "0.2.0",
   "license": "MIT",
-  "description": "Represents a box with text",
+  "description": "Full width banner used for displaying informational messages",
   "keywords": [],
   "author": "Springer Nature",
   "peerDependencies": {
```
Comment (ricky-lau) [inline toolkits/global/packages/global-banner/package.json]: not related to our changes but could bump this to 10.0.0 😁

**Comment by davidpauljunior** [comment]
That's actually a mistake. I was going to leave `marketing` but change `inverted` to `contrast`

**Comment by davidpauljunior** [comment]
Then I was hoping to change across all projects. And then release this as a major version.

File: toolkits/global/packages/global-banner/package.json (hunk 1 above)
Comment (davidpauljunior) [reply toolkits/global/packages/global-banner/package.json]: ^^ global-context 😄

Comment (ricky-lau) [reply toolkits/global/packages/global-banner/package.json]: 🤦‍♂that's what i meant

**Pull request merged**